### DISCUSSION TITLE
第三方插件为啥不起作用

### DISCUSSION THREAD

**Discussion by raincyorg** [BurnHair/docs#118] [技术支持]
![image](https://github.com/BurnHair/docs/assets/35924906/d964b83e-9a0a-4751-b70e-1a298f7319fc)
第三方插件全部打开，但是功能用不了，比如访问网页之类的，是为什么呢？

**Comment by BennyThink** [comment]
<img width="858" alt="image" src="https://github.com/BurnHair/docs/assets/14024832/76d4c5f4-cb86-49e8-a6b5-485b9269027c">

可以啊 你具体场景是什么
